pulpcore/cli/core/context.py

```diff
@@ -73,12 +74,18 @@ class PulpExporterContext(PulpEntityContext):
 
 class PulpExportContext(PulpEntityContext):
     ENTITY = "PulpExport"
-    HREF = "core_pulp_pulp_export_href"
+    HREF = "pulp_pulp_export_href"
     LIST_ID = "exporters_core_pulp_exports_list"
     READ_ID = "exporters_core_pulp_exports_read"
     CREATE_ID = "exporters_core_pulp_exports_create"
     DELETE_ID = "exporters_core_pulp_exports_delete"
 
+    def __init__(self, pulp_ctx: PulpContext) -> None:
+        if not pulp_ctx.has_plugin("pulpcore", min_version="3.10.dev0"):
```

Comment:
It's a little ugly - but so was the problem you solved in pulpcore #1066.

Can you add a ref to that PR as a comment, so we can save the sanity of Future Generations of readers trying to figure out why we're doing this?

(And by 'Future', I mostly mean "me in three weeks")

Reply:
Sure.